frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts

```diff
@@ -219,7 +223,7 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
         }
 
         notebook.status.phase = STATUS_TYPE.TERMINATING;
-        notebook.status.message = 'Preparing to delete the Notebook...';
+        notebook.status.message = 'Preparing to delete the Notebook.';
```

Comment:
I am assuming that any i18n changes will be fixed later, if so, close this comment as resolved.